SourceCode.py

```diff
@@ -1,35 +1,78 @@
 
 
 
+from ast import Pass
+from lib2to3.pgen2.token import OP
 from tkinter import Variable
+from unittest import case
 
 
 class Const : 
-    DATATYPE=["int","float","double","long"]
+    DATATYPE=["int","float","double","long","char"]
     OPERATIONS = ["+","-","*","/","++","--","*=","-=","/=","+="]
-    VARIABLES = []
+    VARIABLES = {}
```

Comment:
names must be small later if you want to define a variable without change value(s), you shode write __ before the name, ex. __data_type, __operations .